src/sentry/integrations/slack/utils/notifications.py

```diff
@@ -163,51 +164,67 @@ def send_incident_alert_notification(
     return success
 
 
+class SlackCommand(Enum):
```

Comment:
Would another potential backlog item be to move commands to the parent class?

Reply:
Although the current set of commands is limited to identity linkage, I think `respond_to_slack_command` wants to remain in `slack/utils/notifications.py` because it was intended to be used for commands in general.

However, your question did give me the idea of moving the values associated with each command into the parent classes, and passing a dataclass here instead of an enum value. I think it's much nicer that way. See https://github.com/getsentry/sentry/pull/74744/commits/3e9926d0739524f657b4219fb659a197c12ba32c